index.js

```diff
@@ -1,5 +1,5 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header").style.color = "green";
```

Comment:
nice. Be careful about doing too many things on one line. It would be more readable in 2 lines

```
const header = document.querySelector("h1#header");
header.style.color = "green";
```

This way you have a variable called header which is now reusable throughout the file.